maqs-base/src/test/java/com/cognizantsoftvision/maqs/base/BaseTestObjectUnitTest.java

```diff
@@ -35,18 +35,17 @@ public void testBaseTestObject1() {
    */
   @Test(groups = TestCategories.FRAMEWORK)
   public void testBaseTestObject2() {
-    final BaseTestObject testObject = this.getTestObject();
+    final BaseTestObject testObject = (BaseTestObject) this.getTestObject();
```

Comment:
Why are we doing all these **BaseTestObject** casts?
As far as I can tell they are not needed.

Reply:
resolved casting issues